R/utils_render_footnotes.R

```diff
@@ -607,7 +607,7 @@ apply_footnotes_to_output <- function(data,
 
 #' @noRd
 set_footnote_marks_row_groups <- function(data,
-                                           context = "html") {
+                                          context = "html") {
```

Comment:
In the future let's get rid of the default values for `context`; we always need these to be explicitly provided.

Reply:
Sounds good!